test/fuzzer/RyanJsonFuzzerParse.c

```
@@ -0,0 +1,126 @@
+		RyanJsonPrintPreallocated(jsonRoot, buf, bufLen / 15, RyanJsonTrue, NULL);
+	}
+
+	free(buf);
+	RyanJsonDelete(jsonRoot);
+	return RyanJsonTrue;
+}
+
+RyanJsonBool_e RyanJsonFuzzerTestMinify(const char *data, uint32_t size)
+{
+	char *buf = (char *)malloc(size + 100);
+	memcpy(buf, data, size);
```

Cppcheck (2.19.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)